From 076f05feb6398d385e06e5b5db605e5a7fe1ee42 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Fri, 23 Feb 2018 19:23:53 -0800
Subject: [PATCH] TOOLS-1983 How to avoid hard failure of most builds on Feb 22
 from GitHub weak crypto removal Reviewed by: Cody Peter Mello
 <cody.mello@joyent.com> Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 targets.json.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/targets.json.in b/targets.json.in
index c9a84cb..1196805 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1035,10 +1035,10 @@ cat <<EOF
     "build_platform": "20151126T062538Z",
     "image_name": "manta-authcache",
     "image_description": "Manta authcache",
-    "image_uuid": "fd2cc906-8938-11e3-beab-4359c665ac99",
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-      "redis-2.4.13",
-      "zookeeper-client-3.4.3"
+      "redis-3.0.5"
     ],
     "repos": [
       {"url": "git@github.com:joyent/mahi.git"}
-- 
2.21.0

